packages/api/src/beacon/routes/validator.ts

```diff
@@ -795,3 +802,7 @@ export function getReturnTypes(): ReturnTypes<Api> {
 function parseBuilderBoostFactor(builderBoostFactorInput?: string | number | bigint): bigint | undefined {
   return builderBoostFactorInput !== undefined ? BigInt(builderBoostFactorInput) : undefined;
 }
+
+function parseSkipRandaoVerification(skipRandaoVerification?: string): boolean {
+  return skipRandaoVerification !== undefined && ["", "true"].includes(skipRandaoVerification) ? true : false;
```

Comment:
we shouldn't check for "true" is is not according to the spec and a bit confusing

Reply:
Empty string is considered a false value, so that's an exception for now. We will encourage the spec to be updated soon and empty string will be left then for backward compatibility.

Reply:
> Empty string is considered a false value

What you mean by that, the code right now considered it as true and also as per spec if flag is present (not undefined) then it should be set to true.

Reply:
In JS  `"" == false ` will evaluate to `true`. So empty string is generally considered a false value, not just in the JS but in different platforms. 

I will open PR to encourage to change the spec so this confusion can be resolved. And as said later on `empty string` will be left as backward compatibility.

Reply:
> In JS "" == false  will evaluate to true. So empty string is generally considered a false value, not just in the JS but in different platforms.

but that's why we explicitly have to parse it here, I don't see the problem

> I will open PR to encourage to change the spec so this confusion can be resolved. And as said later on empty string will be left as backward compatibility.

I don't see how this helps, the proper fix for this is to set `{schema.type : boolean}` in the spec but this is not backward compatible as it would fail schema validation if an empty string is passed.

Reply:
Yes that's what we are discussing earlier as well to suggest to a backward compatible change for the schema to make it flexible. So the schema type remains as the string, but it accepts more values other than just empty string.

Reply:
> but it accepts more values other than just empty string.

but why? This is what I don't get, by stating it's ok to pass true/false here is even more confusing if the schema of the query param is string

I don't see how this helps us or any other client, we can just handle this flag explicitly as done in the PR and be done with it

Reply:
the mistake was to not use boolean schema and it's too late to fix it, maybe if block v4 is being proposed and still has this query param we can push for a schema change

Reply:
Even if we set the schema type to boolean the values we will get will be `true/false` as strings. As the query parameters can only be transmitted/received as strings and then we have to parse those back to actual boolean.

Reply:
I feel this change was more flexible considering future proposed changes to spec, but to get done with this PR had changed it as you felt comfortable.

Reply:
> Even if we set the schema type to boolean the values we will get will be true/false as strings.

But this makes sense and is self explanatory while what you suggested is not well defined as it can't be expressed as part of the schema but has to be explained in the description.

Reply:
> I feel this change was more flexible considering future proposed changes to spec, but to get done with this PR had changed it as you felt comfortable.

For now the aim of the PR is to be spec compliant, if you feel strongly about this we anyways have to add this change to the spec, and can afterwards extend our implementation. Since nobody is sending true as a string for this query param is shouldn't matter.